lib/luhn_cases.rb

```diff
@@ -6,13 +6,15 @@ def name
   end
 
   def work_load
-    %Q(Luhn.valid?("#{input}"))
+    %Q(#{assertion} Luhn.valid?("#{input}"))
```

Comment:
`"#{input}"`

This will work, but it's a good habit to use `input.inspect` which will handle input that contains quotes.

Nice usage of `%Q` though. :+1: